topdown/builtins.go

```diff
@@ -29,18 +31,19 @@ type (
 	// BuiltinContext contains context from the evaluator that may be used by
 	// built-in functions.
 	BuiltinContext struct {
-		Context      context.Context // request context that was passed when query started
-		Seed         io.Reader       // randomization seed
-		Time         *ast.Term       // wall clock time
-		Cancel       Cancel          // atomic value that signals evaluation to halt
-		Runtime      *ast.Term       // runtime information on the OPA instance
-		Cache        builtins.Cache  // built-in function state cache
-		Location     *ast.Location   // location of built-in call
-		Tracers      []Tracer        // Deprecated: Use QueryTracers instead
-		QueryTracers []QueryTracer   // tracer objects for trace() built-in function
-		TraceEnabled bool            // indicates whether tracing is enabled for the evaluation
-		QueryID      uint64          // identifies query being evaluated
-		ParentID     uint64          // identifies parent of query being evaluated
+		Context                context.Context       // request context that was passed when query started
+		Seed                   io.Reader             // randomization seed
+		Time                   *ast.Term             // wall clock time
+		Cancel                 Cancel                // atomic value that signals evaluation to halt
+		Runtime                *ast.Term             // runtime information on the OPA instance
+		Cache                  builtins.Cache        // built-in function state cache
+		PersistentBuiltinCache cache.PersistentCache // built-in function state persistent cache
```

Comment:
I think this comment could be slightly more descriptive. Otherwise it just looks like copy paste. Something like:

```
cross-query built-in function state cache
```

Reply:
Updated